_layouts/default-grid.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Why does there need to be different files for these? There seems to be a lot of repetition between these two.